bluemira/equilibria/fem_fixed_boundary/fem_magnetostatic_2D.py

```diff
@@ -506,32 +545,32 @@ def _plot_current_iteration(
                 f"({i_iter}) " + "$\\Psi_{n}$ error",
                 "seismic",
             )
-            self._add_colorbar(cm, cax[2], "%")
+            _add_colorbar(cm, cax[2], "%")
 
         plt.pause(PLT_PAUSE)
 
-    @staticmethod
-    def _plot_array(
-        ax,
-        points: np.ndarray,
-        array: np.ndarray,
-        title: str,
-        cmap: str,
-        levels: np.ndarray | None = None,
-    ):
-        cm = ax.tricontourf(points[:, 0], points[:, 1], array, cmap=cmap, levels=levels)
-        ax.tricontour(
-            points[:, 0], points[:, 1], array, colors="k", linewidths=0.5, levels=levels
-        )
-
-        ax.set_title(title)
-        return cm
 
-    @staticmethod
-    def _add_colorbar(cm, cax, title):
-        last_axes = plt.gca()
-        ax = cm.axes
-        fig = ax.figure
-        fig.colorbar(cm, cax=cax)
-        cax.set_title(title)
-        plt.sca(last_axes)
+def _plot_array(
```

Comment:
I agree with the change of removing _plot_array and _add_colorbar as @staticmethod (since it wasn’t meaningful in this context). However, it might be better to move these functions outside of the class (into utilities.py?). Could an option be to consider reusing plot_scalar_field?

Reply:
Hmm.. given the use of tricontour, and the fairly trivial nature, I think it is OK to have them as private functions in this module...